platform/jobs/edxEndToEndTestsJob.groovy

```diff
@@ -0,0 +1,70 @@
+package platform
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_WORKER
+
+// The edx-e2e-tests job is run automatically on every deployment of the edx-platform
+// from the gocd pipeline.
+
+job('NEW-edx-e2e-test') {
+
+    description('Run end-to-end tests against the an instance of the edx-platform')
+
+    authorization {
+        blocksInheritance(true)
+        permissionAll('edx')
+    }
+
+    logRotator JENKINS_PUBLIC_LOG_ROTATOR()
+    label(JENKINS_PUBLIC_WORKER)
+    // Disable concurrent builds because the environment in use is a shared
+    // resource, and concurrent builds can cause spurious test results
+    concurrentBuild(false)
+
+    parameters {
+        stringParam('COURSE_ORG', 'ArbiRaees', 'Organization name of the course')
+        stringParam('COURSE_NUMBER', 'AR-1000', 'Course number')
+        stringParam('COURSE_RUN', 'fall', 'Term in which course will run')
+        stringParam('COURSE_DISPLAY_NAME', 'Manual Smoke Test Course 1 - Auto', 'Display name of the course')
+    }
+
+    scm {
+        git {
+            remote {
+                url('https://github.com/edx/edx-e2e-tests.git')
+            }
+            branch('*/master')
+            browser()
+        }
+    }
+
+    wrappers {
+        timeout {
+            absolute()
+        }
+        timestamps()
+        colorizeOutput('gnome-terminal')
+        credentialsBinding {
+            string('BASIC_AUTH_USER', 'BASIC_AUTH_USER')
+            string('BASIC_AUTH_PASSWORD', 'BASIC_AUTH_PASSWORD')
+            string('USER_LOGIN_EMAIL', 'USER_LOGIN_EMAIL')
+            string('USER_LOGIN_PASSWORD', 'USER_LOGIN_PASSWORD')
+        }
+    }
+
+    steps {
+        shell('jenkins/end_to_end_tests.sh')
+    }
+
+    publishers {
+        archiveJunit('reports/*.xml') {
+            allowEmptyResults(false)
+        }
+        archiveArtifacts {
+            pattern('reports/*.xml')
+            pattern('log/*.png')
```

Comment:
couple things:
1. we also need to archive *.log files from the log dir (not just *.png). Probably makes sense to just slurp up anything that is put there.
2. please also archive anything under the 'screenshots' folder